src/zenml/integrations/integration.py

```diff
@@ -42,10 +44,19 @@ class Integration(metaclass=IntegrationMeta):
 
     REQUIREMENTS: List[str] = []
 
+    SYSTEM_REQUIREMENTS: Dict[str, str] = {}
+
     @classmethod
     def check_installation(cls) -> bool:
         """Method to check whether the required packages are installed"""
         try:
+            for req, command in cls.SYSTEM_REQUIREMENTS.items():
+                result = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True, shell=True)
+                if result.returncode!=0:
+                    raise DoesNotExistException(
+                            f"Unable to find linux/unix installation of {req}"
```

Comment:
In my opinion this message should be os - agnostic. Maybe also add a sentence on what the user should do to fix this. Maybe a second sentence like: “Please install the package and retry.”

Reply:
Yes, even I was thinking about this. I will try to use platform to print a more OS-specific message. The only issue is platform.system() prints 'Darwin' for macOS and that might confuse the user at times.

Reply:
Alternativly, you could also just not mention the platform at all. The user already knows his platform and can find the appropriate way to install the package.

Reply:
Yup,  I will do that.